main.js

```diff
@@ -35,6 +35,26 @@ function render() {
   const rootElement = document.querySelector("#root");
   rootElement.innerHTML = "";
 
+  const addPostsFromOutside = document.createElement("input");
+  addPostsFromOutside.setAttribute("type", "button");
+  addPostsFromOutside.setAttribute("value", "Dodaj posty");
+
+  addPostsFromOutside.onclick = () => {
+    fetch("https://jsonplaceholder.typicode.com/posts?userId=1")
+      .then((res) => res.json())
+      .then((data) =>
+        data.forEach((el) => {
+          articles.push({
+            id: articles.length + 1,
+            title: el.title,
+            body: el.body,
+            likesCount: 0,
+          });
+          render();
+        })
+      );
+  };
```

Comment:
It's completely fine! 🚀 But for challange, try move this function to `async`/`await` approach
```suggestion
  addPostsFromOutside.onclick = async () => {
    const result = await fetch("https://jsonplaceholder.typicode.com/posts?userId=1");
    // ToDo...
  };
```